app/src/main/java/com/example/encoratask/service/CharacterService.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.encoratask.service
+
+import android.content.Context
+import com.android.volley.Request
+import com.android.volley.Response
+import com.android.volley.toolbox.StringRequest
+import com.android.volley.toolbox.Volley
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 10/06/21
+ */
+open class CharacterService{
+
+    companion object {
+
+        private val apiUrl = "https://rickandmortyapi.com/api"
+
+        fun getCharacters(context:Context, response:Response.Listener<String>, error: Response.ErrorListener) {
+
+            val url = "$apiUrl/character/"
```

Comment:
you only have one call to api. In this very specific case, there's no reason to part this URL with "$apiUrl/character/"

Reply:
I understand but for future ads it gonna solve a lot of, and for GOOD practices I was thinkg was OK